package.json

```diff
@@ -56,30 +56,32 @@
     "@formatjs/intl-pluralrules": "^1.5.4",
     "@formatjs/intl-relativetimeformat": "^4.5.11",
     "@momentum-ui/react": "^23.21.5",
-    "@webex/common": "1.80.271",
-    "@webex/common-timers": "1.80.271",
-    "@webex/helper-html": "1.80.271",
-    "@webex/http-core": "1.80.271",
-    "@webex/internal-plugin-conversation": "1.80.271",
-    "@webex/internal-plugin-encryption": "1.80.271",
-    "@webex/internal-plugin-feature": "1.80.271",
-    "@webex/internal-plugin-flag": "1.80.271",
-    "@webex/internal-plugin-mercury": "1.80.271",
-    "@webex/internal-plugin-metrics": "1.80.271",
-    "@webex/internal-plugin-presence": "1.80.271",
-    "@webex/internal-plugin-search": "1.80.271",
-    "@webex/internal-plugin-team": "1.80.271",
-    "@webex/internal-plugin-user": "1.80.271",
-    "@webex/internal-plugin-wdm": "1.80.271",
-    "@webex/plugin-authorization": "1.80.271",
-    "@webex/plugin-logger": "1.80.271",
-    "@webex/plugin-meetings": "1.80.271",
-    "@webex/plugin-people": "1.80.271",
-    "@webex/plugin-phone": "1.80.271",
-    "@webex/plugin-rooms": "1.80.271",
-    "@webex/storage-adapter-local-forage": "1.80.271",
-    "@webex/storage-adapter-local-storage": "1.80.271",
-    "@webex/webex-core": "1.80.271",
+    "@webex/common": "1.98.2",
+    "@webex/common-timers": "1.98.2",
+    "@webex/components": "^1.45.3",
+    "@webex/helper-html": "1.98.2",
+    "@webex/http-core": "1.98.2",
+    "@webex/internal-plugin-conversation": "1.98.2",
+    "@webex/internal-plugin-encryption": "1.98.2",
+    "@webex/internal-plugin-feature": "1.98.2",
+    "@webex/internal-plugin-flag": "1.98.2",
+    "@webex/internal-plugin-mercury": "1.98.2",
+    "@webex/internal-plugin-metrics": "1.98.2",
+    "@webex/internal-plugin-presence": "1.98.2",
+    "@webex/internal-plugin-search": "1.98.2",
+    "@webex/internal-plugin-team": "1.98.2",
+    "@webex/internal-plugin-user": "1.98.2",
+    "@webex/internal-plugin-wdm": "1.98.2",
+    "@webex/plugin-authorization": "1.98.2",
+    "@webex/plugin-logger": "1.98.2",
+    "@webex/plugin-meetings": "1.98.2",
+    "@webex/plugin-people": "1.98.2",
+    "@webex/plugin-phone": "1.98.2",
```

Comment:
Can we remove this now? We can also remove the import [here](https://github.com/webex/react-widgets/blob/master/packages/node_modules/@webex/react-redux-spark/src/sdk.js#L6)

Reply:
I will give it a shot and see if it breaks anything.